### PR TITLE
Milestone 1 App initial setup

### PR DESCRIPTION
**Pull request by Diegodsha**
- Use create-react-app to initialize a new React app 
- Delete any unnecessary file or asset
- Add big.js using npm.
- Follow the instructions in create-react-app-buildpack to deploy the app to Heroku

### CODE REVIEW

**Review by trekab — COMMENTED** [review]
### Invalid! :stop_sign: 
Hi @Diegodsha! Unfortunately, I have to invalidate your PR because you didn't follow the expected [Git Flow](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59882084/) on this project.

_Please make the necessary changes and submit another code review request._
__[Treasure Kabareebe](https://trekab.github.io)__

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @Diegodsha  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**1.** Opps, you forgot to add [prop-types](https://www.npmjs.com/package/prop-types) to `package.json`
**2.** Let's make your repo looks professional by applying all the 3 basic rules, kindly update your `README` file and remove unnecessary content.
Checkout out the below link for help, feel free to follow this [template](https://github.com/microverseinc/readme-template) from pathwright

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: package.json [hunk 1]
```diff
@@ -35,5 +35,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
```
Comment (bellom) [inline package.json]: @Diegodsha, Good job adding this, but to make this work properly, you need to add it to `dependencies`, not `devDependencies `

I trust you can do this before merging. 👀 on you.

**Review by bellom — APPROVED** [review]
## Hi @Diegodsha  :clap: 

**Great work trying to make all the necessary corrections from the previous review.**

### [Optional] 
This is marked ad optional because I missed it in the first review, kindly fix this if you don't your next milestone would be marked as invalid.
**You need to set up linter.**
checkout this [linters-config#troubleshooting](https://github.com/microverseinc/linters-config#troubleshooting) for help. Example below;
image
If you are not having this check on your `Github` page, kindly visit :point_right:[ linter set up](https://github.com/microverseinc/linters-config/tree/master/react-redux)

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best**

**Pull request merged**